packages/lisk-transactions/src/utils/get_address_and_public_key_from_recipient_data.js

```diff
@@ -18,19 +18,7 @@ const getAddressAndPublicKeyFromRecipientData = ({
 	recipientId,
 	recipientPublicKey,
 }) => {
```

Comment:
I don't think this function adds much value anymore. Can we remove it?

Reply:
@willclarktech we are still using it in type o transactions.

Reply:
@willclarktech are you suggesting to use `cryptography.getAddressFromPublicKey` directly in the tx function ?

Reply:
Yes.